components/atoms/Breadcrumb.js

```diff
@@ -7,7 +7,7 @@ import Link from "next/link";
 export function Breadcrumb(props) {
   return (
     <nav aria-label="breadcrumbs">
-      <ul className="block text-custom-blue-dark text-base font-body">
+      <ul className="block text-custom-blue-dark text-base font-body -my-4 -ml-4">
```

Comment:
Breadcrumb spacing looks a little funky. Doing something like this brings it closer to what's on Canada.ca currently:

```jsx
    <nav aria-label="breadcrumbs">
      <ul className="block text-custom-blue-dark text-base font-body -my-4 -ml-4">
        <li className="inline-block min-w-0 max-w-full truncate px-1">
          <Link href="https://www.canada.ca/">
            <a className="text-sm hover:text-custom-blue-link visited:text-purple-700 underline">
              Canada.ca
            </a>
          </Link>
        </li>

        {props.items
          ? props.items.map((item, key) => {
              return (
                <li
                  key={key}
                  className="inline-block min-w-0 max-w-full truncate"
                >
                  <span className="inline-block align-middle text-gray-breadcrumb icon-cheveron-right mr-4" />
                  <Link href={item.link}>
                    <a className="text-sm hover:text-canada-footer-hover-font-blue text-canada-footer-font visited:text-purple-700 underline">
                      {item.text}
                    </a>
                  </Link>
                </li>
              );
            })
          : null}
      </ul>
    </nav>
```